### PR TITLE
Fix image name in the build/test hack scripts

### PR DESCRIPTION
**Pull request by jhadvig**
+ adding naming explanation

### CODE REVIEW

**Comment by jhadvig** [comment]
@mfojtik thoughts ?

File: hack/build.sh [hunk 1]
```diff
@@ -17,7 +17,12 @@ else
 fi
 
 for dir in ${dirs}; do
-  IMAGE_NAME=nodejs-${dir}-${OS}
+  # Image name structure: 
+  # 1. Platform - nodejs
+  # 2. Platform version(without dots)
```
Comment (mfojtik) [inline hack/build.sh]: This should be described in README afaik.

**Comment by jhadvig** [comment]
@mfojtik updated

**Comment by mfojtik** [comment]
LGTM

**Pull request merged**